tests/src/testeas.hh

```diff
@@ -10,8 +10,9 @@
 
 #include <ikarus/finiteelements/mechanics/enhancedassumedstrains.hh>
 
-template <typename DisplacementBasedElement>
-struct ElementTest<Ikarus::EnhancedAssumedStrains<DisplacementBasedElement>>
+template <typename FE>
+requires(FE::template hasSkill<Ikarus::EnhancedAssumedStrainsPre::Skill>())
```

Comment:
maybe we can register the Skill globally then its available as `Ikarus::EnhancedAssumedStrainsSkill` but this could get quickly out of hand with a lot of different skills